integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesEvents.java

```diff
@@ -1213,6 +1214,15 @@ private void scaleDomainAndVerifyCompletedEvent(int replicaCount, String testTyp
         + "{\"op\": \"replace\", \"path\": \"/spec/clusters/0/replicas\", \"value\": " + replicaCount + "}" + "]");
     assertTrue(patchDomainCustomResource(domainUid, domainNamespace1, patch, V1Patch.PATCH_FORMAT_JSON_PATCH),
         "Failed to patch domain");
+    if (testType != null) {
+      if (testType.equals("scale up")) {
```

Comment:
Are there other values of testType? I think this should be a enum rather than a String.

Reply:
switched to enum